src/app/project/project.component.html

```diff
@@ -138,7 +138,7 @@ <h1>Find EA Projects in British Columbia</h1>
           <td class="proj-col proponent"><span>{{item.proponent.name}}</span></td>
           <td class="proj-col type"><span>{{item.type}}</span></td>
           <td class="proj-col status"><span>{{item.eacDecision}}</span></td>
-          <td class="proj-col date"><span *ngIf='item.decisionDate'>{{item.decisionDate | date:'dd-MMM-yyyy'}}</span>
+          <td class="proj-col date"><span *ngIf='item.decisionDate'>{{item.decisionDate | date:'yyyy-MM-dd'}}</span>
```

Comment:
We should be using the built-in Angular date formats, for consistency (and maintainability). See [this example](https://github.com/bcgov/eao-public/blob/aeaa6bb892a538e2a96ca27d1373705b84d93920/src/app/project-detail/project-detail.component.html#L84) as reference.